composer.json

```diff
@@ -1,12 +1,9 @@
 {
   "name": "PHP Quickstart",
```

Comment:
You should probably update the name of this project while we're at it.

Reply:
Changing to "PHP Blog Example", feels like it encapsulates the goal of the project